src/ui/ui-kit/Frame/styles.module.css

```diff
@@ -0,0 +1,20 @@
+.frame {
+  background-color: var(--frame-background-color);
+  border: 2px solid var(--frame-border-color, var(--neutral-200));
+  border-radius: 16px;
+  padding: 8px;
+}
+
+@media (hover: hover) {
+  .interactive:enabled:hover {
+    background-color: var(--neutral-100);
+  }
+}
+
+.interactive:enabled:focus {
```

Comment:
It looks like we show outline on focus anyway. Do we need to change background color too?

Reply:
🤔 i think you're right, we do not